internal/configs/annotations_test.go

```diff
@@ -163,6 +166,58 @@ func TestMergeMasterAnnotationsIntoMinion(t *testing.T) {
 	}
 }
 
+func TestParseProxySetHeaderInputString(t *testing.T) {
+	t.Parallel()
+	headers1 := []string{"X-Forwarded-For"}
+	headers2 := []string{"ABC"}
+	headers3 := []string{"Test"}
+
+	headers := [][]string{headers1, headers2, headers3}
+	for _, header := range headers {
+		err := ParseProxySetHeader(header)
+		if err != nil {
+			t.Errorf("want nil, got error on valid input: %+v", header)
+		}
+	}
+}
+
+func TestParseProxySetHeaderInvalidInputString(t *testing.T) {
+	t.Parallel()
+	headers1 := []string{"X-Forwarded-For1"}
```

Comment:
Why is this header invalid? I believe $http_x_forwarded_for1 would be a valid nginx variable

Reply:
The header cannot have any numbers in it so therefore it breaks. If the user wants the value to have a number, that is okay.